### DISCUSSION TITLE
Bundle size server

### DISCUSSION THREAD

**Discussion by hjaber** [MasterKale/SimpleWebAuthn#473] [General]
First of all, thank you for the hard work you've put into this incredible project. I have successfully set it up on my own server, and everything is running smoothly.

I have noticed, however, that some individual imports are showing up as greater than 50kb in bundle size according to my VS Code extension. This seemed surprising, considering that it's almost the same size as the entire package. I was wondering if this could be confirmed or if it's possible that my extension is misreading the data.

<img width="698" alt="image" src="https://github.com/MasterKale/SimpleWebAuthn/assets/41503068/84334a11-7e5f-40eb-87ba-90ec30385bf0">

https://bundlephobia.com/package/@simplewebauthn/server@8.3.5

**Comment by MasterKale** [comment]
I'm not sure I understand what the problem is exactly. This library is server-centric so I've not paid as much attention to the size of **@simplewebauthn/server** as I do **@simplewebauthn/browser**. Is this making it hard to deploy in edge runtimes because of deployment file size constraints?

**Comment by hjaber** [comment]
No issue deploying at the edge so far. 

But I thought it wasn't optimal that a single function import was 55kb which is the size of the entire server library.
